CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 Wazuh commit: https://github.com/wazuh/wazuh/commit/88bf15d2cbb2040e197e34a94dda0f71f607afad \
 Release report: https://github.com/wazuh/wazuh/issues/14827
 
+### Added
+
+- Analysisd - add new test to check analysisd socket properties ([#2405](https://github.com/wazuh/wazuh-qa/pull/2405))
+
```

Comment:
It has the project `4.3.9` but in the documentation, you add it on `4.3.8`. 

Why you don't respect the new syntax defined? it  shouldn't be like this: 

> Add Analysisd IT - new test to check analysisd socket properties ([#2405](https://github.com/wazuh/wazuh-qa/pull/2405)) \- (Framework + Tests)

Reply:
This behavior is expected since only the branch was generated but migrated with the same changes.